styles/tokens/input.json

```diff
@@ -132,29 +123,104 @@
       "file": "settings",
       "themable": true
     },
-    "focus-border-color": {
-      "comment": "The border color on Inputs when focused.",
-      "value": "{purple.value}",
-      "file": "settings",
-      "themable": true
-    },
-    "error-border-color": {
-      "comment": "The border color on Inputs when there is an error.",
-      "value": "{orange.value}",
-      "file": "settings",
-      "themable": true
+    "focus": {
+      "border-color": {
```

Comment:
Let's take a look at this -- we got rid of all borders on inputs so this might not be necessary